### PR TITLE
Do not require proxy

### PR DESCRIPTION
**Pull request by annawoodard**

### CODE REVIEW

**Comment by annawoodard** [comment]
I've tested this successfully with the following configuration: `/afs/crc.nd.edu/user/a/awoodard/lobster-dev/lobster/examples/simple.py`.

**Comment by annawoodard** [comment]
@matz-e: This isn't how we usually want to do things, but I'm going to go ahead and merge now so that we can get Anna going with local running today. If you have a chance, please do a post-merging review and I will implement your comments on the master.

File: lobster/core/config.py [hunk 1]
```diff
@@ -195,6 +200,7 @@ def __init__(self,
             log_level=2,
             payload=10,
             renew_proxy=True,
+            require_proxy=False,
```
Comment (matz-e) [inline lobster/core/config.py]: I don't think this should be False by default!

Comment (annawoodard) [inline lobster/core/config.py]: @matz-e: Good point. I'll change the default to True. But unless you have objections, I'll leave the handling for the `file` protocol in.

Comment (matz-e) [inline lobster/core/config.py]: Sounds good. Is the `copy_output` bit already in place? I only see changes to the `check_output`? Will be a few more minutes before I have more time

Comment (annawoodard) [inline lobster/core/config.py]: `copy_output` doesn't need to change as long as the `file` protocol is handled by `check_output`.

**Pull request merged**